Added detailed code on how to implement aws s3 and firebase with image picker. (#12924)
expo/expo: docs/pages/versions/unversioned/sdk/imagepicker.md

}
```

## Using ImagePicker with AWS S3

> Please refer to our example [here](https://github.com/expo/examples/tree/master/with-aws-storage-upload). Make sure you follow Amplify's docs to set your project up correctly.

Reviewer: ```suggestion
Please refer to the [with-aws-storage-upload example](https://github.com/expo/examples/tree/master/with-aws-storage-upload). Follow [Amplify docs](https://docs.amplify.aws/) to set your project up correctly.
```

## Using ImagePicker with Firebase

> Please refer to our example [here](https://github.com/expo/examples/tree/master/with-firebase-storage-upload). Make sure you follow Firebases' docs to set your project up correctly.

Reviewer: ```suggestion
Please refer to the [with-firebase-storage-upload example](https://github.com/expo/examples/tree/master/with-firebase-storage-upload). Make sure you follow the ["Using Firebase"](/guides/guides/using-firebase.md) docs to set your project up correctly.
```
